docs/logging-configuration.asciidoc

```diff
@@ -0,0 +1,105 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/ruby[elastic.co]
+endif::[]
+
+[[logging-configuration]]
+== Logging configuration
+
+The Ruby agent allows you to set logging options as well as configure trace/log correlation.
+Please see the <<configuration,configuration documentation>> to learn how to set <<config-logger,a logger>>, <<config-log-path,
+a log path>> and <<config-log-level,log level>>.
+
+[float]
+[[trace-log-correlation]]
+==== Trace/log correlation
+
+Trace/log correlation can be set up in three different ways.
+
+[float]
+[[rails-tagged-logging]]
+==== Rails TaggedLogging
+
+Rails applications configured with an `ActiveSupport::TaggedLogging` logger can append the correlation IDs to log output.
+Add the following to the Rails environment configuration file:
```

Comment:
I think the stock `rails new` generated app has a mention of `log_tags` somewhere. Perhaps we should use that in our example somehow?

Reply:
The stock `rails new` app includes this line in `production.rb`
```
  # Prepend all log lines with the following tags.
  config.log_tags = [ :request_id ]
```
Are you suggesting I mention that file specifically and update that line ⬆️ ?

Reply:
Yes, that was the one I was thinking of. Something like "for example in your `config/environments/production.rb` put this blah blah":
```ruby
# Prepend all log lines with the following tags.
  config.log_tags = [ :request_id, proc { ElasticAPM.log_ids } ]
```

Reply:
https://github.com/elastic/apm-agent-ruby/pull/527/commits/91aab6afd60139c74a6dfbf404d888d72a4672ab